File: 8Kyu/JavaScript/simple_multiplication.js
// This kata is about multiplying a given number by eight if it is an even number and by nine otherwise.

function simpleMultiplication(number) {
    if (number % 2 == 0) {
        return number * 8
    } else {
        return number * 9
    }
}

function simpleMultiplication(number) {
    return number * (number % 2 ? 9 : 8)
}

function simpleMultiplication(number) {
    return number % 2 == 0 ? number * 8 : number * 9
}

const simpleMultiplication = number => number % 2 == 0 ? number * 8 : number * 9

File: 8Kyu/Python/simple_multiplication.py
#  This kata is about multiplying a given number by eight if it is an even number and by nine otherwise.

def simple_multiplication(number):
    if number % 2 == 0:
        return number * 8
    else:
        return number * 9

def simple_multiplication(number):
    return number * 8 if number % 2 == 0 else number * 9

simple_multiplication = lambda number : number * 8 if number % 2 == 0 else number * 9